proxy/src/main/java/com/velocitypowered/proxy/config/AnnotatedConfig.java

```diff
@@ -144,45 +156,50 @@ private String toString(Object value) {
             if (listValue.isEmpty()) {
                 return "[]";
             }
+
             StringBuilder m = new StringBuilder();
             m.append("[");
+
             for (Object obj : listValue) {
                 m.append(System.lineSeparator()).append("  ").append(toString(obj)).append(",");
             }
+
             m.deleteCharAt(m.length() - 1).append(System.lineSeparator()).append("]");
             return m.toString();
         }
+
         if (value instanceof Enum) {
             value = value.toString();
         }
+
         if (value instanceof String) {
             String stringValue = (String) value;
             if (stringValue.isEmpty()) {
                 return "\"\"";
             }
             return "\"" + stringValue.replace("\n", "\\n") + "\"";
         }
+
         return value != null ? value.toString() : "null";
     }
 
     /**
-     * Saves lines to file
+     * Wrties list of strings to file
      *
-     * @param lines Lines to save
-     * @param to A path of file where to save lines
-     * @throws IOException if lines is empty or was error during saving
+     * @param lines list of strings to write
+     * @param to Path of file where lines should be written
+     * @throws IOException if error occurred during writing
      */
     public static void saveConfig(List<String> lines, Path to) throws IOException {
-        if (lines.isEmpty()) {
```

Comment:
Removing braces from if statements is considered bad practice and has led to severe flaws in software before - [a good example is Apple's "goto fail" bug](https://www.imperialviolet.org/2014/02/22/applebug.html) which arose from one line if statements not using braces. Please _always_ use braces with if statements.

Reply:
Ouch. I left that in accidentally. Will revert this.